mmdet3d/models/dense_heads/centerpoint_head.py

```diff
@@ -548,19 +554,31 @@ def get_targets_single(self, gt_bboxes_3d, gt_labels_3d):
                     ind[new_idx] = y * feature_map_size[0] + x
                     mask[new_idx] = 1
                     # TODO: support other outdoor dataset
-                    vx, vy = task_boxes[idx][k][7:]
-                    rot = task_boxes[idx][k][6]
-                    box_dim = task_boxes[idx][k][3:6]
-                    if self.norm_bbox:
-                        box_dim = box_dim.log()
-                    anno_box[new_idx] = torch.cat([
-                        center - torch.tensor([x, y], device=device),
-                        z.unsqueeze(0), box_dim,
-                        torch.sin(rot).unsqueeze(0),
-                        torch.cos(rot).unsqueeze(0),
-                        vx.unsqueeze(0),
-                        vy.unsqueeze(0)
-                    ])
+                    if self.with_velocity:
+                        vx, vy = task_boxes[idx][k][7:]
+                        rot = task_boxes[idx][k][6]
```

Comment:
line 559 to 562 can be moved out of the if-else

Reply:
that's right